packages/components/src/components/accordion-item/accordion-item.scss

```diff
@@ -155,6 +166,19 @@
       ease-in-out;
     @include includes.word-break();
   }
+  .has-actions {
+    gap: var(--calcite-internal-accordion-item-action-slot-spacing);
+  }
+  .actions-start.has-actions {
+    padding-block: var(--calcite-internal-accordion-item-action-slot-spacing)
```

Comment:
The styling for `padding-block` is duplicated between rule sets.

Reply:
Some kind of codemod did this.  I wrote it as a single `padding:` rule.  Any way to disable that?